pkg/apis/garden/validation/validation_test.go

```diff
@@ -1739,6 +1739,261 @@ var _ = Describe("validation", func() {
 				})
 			})
 		})
+
+		// BEGIN PACKET
+		Context("tests for Packet cloud profiles", func() {
+			var (
+				fldPath         = "packet"
+				packetProfile *garden.CloudProfile
+			)
+
+			BeforeEach(func() {
+				packetProfile = &garden.CloudProfile{
+					ObjectMeta: metadata,
+					Spec: garden.CloudProfileSpec{
+						Packet: &garden.PacketProfile{
+							Constraints: garden.PacketConstraints{
+								DNSProviders: dnsProviderConstraint,
+								Kubernetes:   kubernetesVersionConstraint,
+								MachineImages: []garden.PacketMachineImage{
+									{
+										Name: "Container Linux - Stable",
+										ID:   "d61c3912-8422-4daf-835e-854efa0062e4",
+									},
+								},
+								MachineTypes: machineTypesConstraint,
+								VolumeTypes:  volumeTypesConstraint,
+								Zones: zonesConstraint,
+							},
+						},
+					},
+				}
+			})
+
+			It("should not return any errors", func() {
+				errorList := ValidateCloudProfile(packetProfile)
+
+				Expect(len(errorList)).To(Equal(0))
+			})
+
+			It("should forbid ca bundles with unsupported format", func() {
+				packetProfile.Spec.CABundle = makeStringPointer("unsupported")
+
+				errorList := ValidateCloudProfile(packetProfile)
+
+				Expect(len(errorList)).To(Equal(1))
```

Comment:
Better use
```golang
Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
	"Type":  Equal(field.ErrorTypeInvalid),
	"Field": Equal("spec.caBundle"),
}))))
```

(from https://github.com/gardener/gardener/blob/master/pkg/apis/garden/validation/validation_test.go#L374-L377)

Reply:
Apply this for all new tests.

Reply:
This has not yet been resolved, right? Please do.

Reply:
Nope, working on it now.

Reply:
And stuck. Message in slack channel, but, how do I compare `errorList` to multiple at once? E.g.

```go
                                        Expect(len(errorList)).To(Equal(2))
                                        Expect(*errorList[0]).To(MatchFields(IgnoreExtras, Fields{
                                                "Type":  Equal(field.ErrorTypeRequired),
                                                "Field": Equal(fmt.Sprintf("spec.%s.constraints.volumeTypes[0].name", fldPath)),
                                        }))
                                        Expect(*errorList[1]).To(MatchFields(IgnoreExtras, Fields{
                                                "Type":  Equal(field.ErrorTypeRequired),
                                                "Field": Equal(fmt.Sprintf("spec.%s.constraints.volumeTypes[0].class", fldPath)),
                                        }))
```

becomes what?

Reply:
```golang
Expect(errorList).To(ConsistOf(PointTo(MatchFields(IgnoreExtras, Fields{
	"Type":  Equal(field.ErrorTypeRequired),
	"Field": Equal(fmt.Sprintf("spec.%s.constraints.volumeTypes[0].name", fldPath)),
})),
	"Type":  Equal(field.ErrorTypeRequired),
	"Field": Equal(fmt.Sprintf("spec.%s.constraints.volumeTypes[0].class", fldPath)),
))
```

Reply:
OK will try it

Reply:
I think I got them all for Packet.